scripts/import-sounds-to-mongo.ts

```diff
@@ -1,13 +1,13 @@
 import { MongoClient } from 'mongodb';
-import { SoundDocument } from 'botman-sounds/src/sound-document';
-import { SoundFile } from 'botman-sounds';
 import { promises as fs } from 'fs';
+import { SoundFile } from 'botman-sounds';
+import { SoundDocument } from 'botman-sounds/src/sound-document';
 
 // Use this script to import sounds from a folder into a mongo database.
 // Doesn't check for or clear existing documents, so running it twice will produce duplicates.
 
 const connectionUri = 'mongodb://localhost:27017';
-const soundsDirectory = '../bot/sounds';
+const soundsDirectory = 'sounds';
 
 function splitFileName(name: string): SoundFile {
   const fullName = name.toLowerCase();
```

Comment:
```suggestion
  const fullName = name;
```

You're not doing this in the blob stuff so any sound that had caps gets `404` when it goes to look for it.

Reply:
Come to think of it I should probably just combine them into one "import" script now.